File: secret_handshake.rs
const VEC : [&str; 4] = ["wink","double blink","close your eyes","jump"];
pub fn actions(n: u8) -> Vec<&'static str> {
    let mut ans = Vec::new();
    let mut tmp = 1;
    for i in 0..=4{
        match n & tmp {
            0 => {
                tmp = tmp << 1;
            }
            1.. => {
                if i == 4 {
                    ans.reverse();
                    break;
                }
                ans.push(VEC[i]);
                tmp = tmp << 1;
            }
        }
    }
    ans
}
/// there's also a pretty neat solution from babahop on exercism

File: solutions/python/secret-handshake/1/secret_handshake.py
def commands(binary_str):
    answer = []
    if binary_str[4] == '1':
        answer.append("wink")
    if binary_str[3] == '1':
        answer.append("double blink")
    if binary_str[2] == '1':
        answer.append("close your eyes")
    if binary_str[1] == '1':
        answer.append("jump")
    if binary_str[0] == '1':
        answer.reverse()
    return answer
        
    
